backend-api/src/functions/common/session/Session.ts

```diff
@@ -0,0 +1,61 @@
+import { AttributeValue } from "@aws-sdk/client-dynamodb";
+
+export enum SessionState {
+  AUTH_SESSION_CREATED = "ASYNC_AUTH_SESSION_CREATED",
+  BIOMETRIC_TOKEN_ISSUED = "ASYNC_BIOMETRIC_TOKEN_ISSUED",
+  BIOMETRIC_SESSION_FINISHED = "ASYNC_BIOMETRIC_SESSION_FINISHED",
+  RESULT_SENT = "ASYNC_RESULT_SENT",
+}
+
+type CommonSessionKeys = {
+  sessionId: string;
+  createdAt: number;
+  timeToLive: number;
+  clientId: string;
+  govukSigninJourneyId: string;
+  issuer: string;
+  clientState: string;
+  subjectIdentifier: string;
+  redirectUri?: string;
+};
+
+export type AuthSessionCreatedSession = CommonSessionKeys & {
+  sessionState: SessionState.AUTH_SESSION_CREATED;
+};
+
+export type BiometricTokenIssuedSession = CommonSessionKeys & {
+  sessionState: SessionState.BIOMETRIC_TOKEN_ISSUED;
+  documentType: string;
+  accessToken: string;
```

Comment:
More for the back of your mind and not related to this spike - we shouldn't save the accessToken in the database - it's a short-lived token and I can't think why we'd ever need it!

Reply:
Good point, I hadn't stopped to consider where it would actually be used again. One to update on the [TD](https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3957719041/Strategic+App+ID+Check+Document+Selection+in+app) where it came from I suppose - perhaps as part of a more general review of the session schema?

Reply:
Yes! It's on my list to update that TD having spoken to Darren